include/OpenColorIO/OpenColorIO.h

```diff
@@ -2432,6 +2433,13 @@ class OCIOEXPORT Baker
     /// Set the target device colorspace for the LUT.
     void setTargetSpace(const char * targetSpace);
 
+    const char * getDisplay() const;
+
+    const char * getView() const;
+
+    /// Set the display and view to apply during the baking.
+    void setDisplayView(const char * display, const char * view);
+
```

Comment:
I think we still need Python bindings for these bad boys.

Reply:
Good catch, added it now.